raiden/tests/integration/test_matrix_transport.py

```diff
@@ -771,3 +785,403 @@ def make_unsigned_balance_proof(nonce):
     )
     transport.stop()
     transport.get()
+
+
+@pytest.mark.parametrize('privacy_settings', [
+    [True, True, 'invite'],
+    [True, False, 'invite'],
+    [False, True, 'public'],
+    [False, False, 'public'],
+])
```

Comment:
Conflating the room invite type into the `privacy_settings` fixture isn't very clean IMO.
Please use a separate parameter for that.